gvfs-helper-client.c

```diff
@@ -168,28 +168,6 @@ static int gh_client__send__objects_prefetch(struct child_process *process,
 	return 0;
 }
 
-/*
- * Verify that the pathname found in the "odb" response line matches
- * what we requested.
- *
- * Since we ALWAYS send a "--shared-cache=<path>" arg to "gvfs-helper",
- * we should be able to verify that the value is what we requested.
- * In particular, I don't see a need to try to search for the response
- * value in from our list of alternates.
- */
-static void gh_client__verify_odb_line(const char *line)
-{
-	const char *v1_odb_path;
-
-	if (!skip_prefix(line, "odb ", &v1_odb_path))
-		BUG("verify_odb_line: invalid line '%s'", line);
-
-	if (!gh_client__chosen_odb ||
-	    strcmp(v1_odb_path, gh_client__chosen_odb->path))
-		BUG("verify_odb_line: unexpeced odb path '%s' vs '%s'",
```

Comment:
@jeffhostetler @dscho I hit this BUG during _one_ of the functional tests near the end of the [Scalar release build process](https://dev.azure.com/mseng/Scalar/_build/results?buildId=14099960&view=logs&j=de015ea9-9ec8-581a-12fc-2741c13e533e&t=9dce4694-c742-5e62-10d3-fa0894667d5b&l=247) so I'm going to cherry-pick this on top of `vfs-2.30.0`.